libs/constants/src/regexp/user-alias.regexp.ts

```diff
@@ -0,0 +1 @@
+export const USER_ALIAS_REGEXP = /^(?![\s028u{}]*$).+/;
```

Comment:
That's pretty weird regex. For example, nickname "u2u822" would be discarded, even though it is pretty random, it's not invalid

Reply:
Just to follow up on this - this was a regex I wrote, based on a SO answer (which I tried to dig up just now but failed). The intention is to match any string that's not *entirely* whitespace. I recommend regexr.com for if you're struggling to understand a regex, see [regexr.com/816p3](this one) for a breakdown of what this one is doing.